config/axios.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios');
```

Comment:
I think it would be better in a folder of helpers, what do you think?

Reply:
It is true, reading the description of the helpers in the tech guide I see that it meets the main characteristics, does not contain business logic and could be implemented in other projects.